tests/controllers/test_rss_controller.py

```diff
@@ -142,9 +136,52 @@ def test_try_parse_feed_with_beautiful_soup(feed_input: str, is_url: bool, enabl
 @pytest.mark.parametrize("enable_cache", [True, False])
 def test_enable_disable_cache(enable_cache):
     feed_input = "https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml"
-    rss_controller = RSSController(feed_input=feed_input, enable_cache=enable_cache)
+    controller = RSSController(feed_input=feed_input, enable_cache=enable_cache)
 
     if enable_cache:
-        assert rss_controller._session
+        assert controller._session
     else:
-        assert not rss_controller._session
+        assert not controller._session
+
+
+def test_parse_feeds():
+    feed_input = "https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml"
```

Comment:
I'm not sure I like the idea of hitting an external site in our unit tests, can we just capture one of these calls and store the data for use in the parsing test?

Reply:
Removed external sites.